portfolio-alex.css

```
@@ -0,0 +1,59 @@
+*{
+box-sizing:border-box;
+}
+
+body{
+	margin:0;
+	padding:0;
+}
+
+.topheader{
+	width:100%;
+	height:300px;
+}
+	
+.logo{
+	background-color:#E6E6FA;
+	width:100%;
+	height:200px;
+	display:flex;
+	justify-content:center;
+	align-items:center;
+	text-align:center;
+}
+
+.logotext{
+	font-size:30px;
+	font-family:spartan;
+	width:60%;
+	margin:auto;
+}
+
+.navlinks{
+	border:none;
+	font-size:25px;
+	padding:10px;
+	line-height:50px;
+	display:flex;
+	justify-content:center;
+	align-items:center;
+	height:100px;
+	font-family:spartan;
```

It's a good idea to wrap the font name in quotes.
```suggestion
	font-family: "spartan";
```